modules/blocks/verify.js

```diff
@@ -231,6 +268,18 @@ Verify.prototype.processBlock = function (block, broadcast, cb, saveBlock) {
 	}
 
 	async.series({
+		addBlockProperties: function (seriesCb) {
```

Comment:
`Block.prototype.create` calls to `objectNormalize`, there are no missed properties, therefore no need to perform addBlockProperties logic. Same for `loadBlocksOffset`, with data from DB.
`receiveBlock` will need it, so we can add it to public logic.block and call it from `transport` and `verify` to avoid  one function with two logics.